Formula/kumo.rb

```diff
@@ -0,0 +1,20 @@
+class Kumo < Formula
+  desc "Word Clouds in Java"
```

Comment:
why is the C in Clouds capitalized

Reply:
I guess upstream likes doing that :see_no_evil:

Reply:
Yeah I left it because it mimics upstream 😸. It could be changed but, depends how much you care I guess.

Reply:
I'm going to assume Word Clouds are now a thing like The Internet and will eventually be lower case again like the internet.

Reply:
That's a very good question! haha I have never thought about it. I have spent way more of my professional career than I care to admit working with word clouds. :)